sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -88,6 +88,8 @@ void AddCustomerPolicies(HttpPipelinePosition position)
 
             var sanitizer = new HttpMessageSanitizer(diagnostics.LoggedHeaderNames.ToArray(), diagnostics.LoggedQueryParameters.ToArray());
 
+            responseClassifier ??= new ResponseClassifier() { MessageSanitizer = sanitizer };
```

Comment:
Is there a reason why we create a new classifier here as opposed to using a singleton?

Reply:
I'm planning to change this per a bug @pakrym identified.